src/stream/gst/pipeline_runner.rs

```diff
@@ -8,18 +8,18 @@ pub struct Pipeline {
     pub description: String,
 }
 
-#[derive(Default)]
+// #[derive(Default)]
 pub struct PipelineRunner {
     pub pipeline: Pipeline,
 }
 
-impl Default for Pipeline {
-    fn default() -> Self {
-        Pipeline {
-            description: "videotestsrc pattern=ball ! video/x-raw,width=640,height=480 ! videoconvert ! x264enc bitrate=5000 ! video/x-h264, profile=baseline ! rtph264pay ! udpsink host=0.0.0.0 port=5600".into(),
-        }
-    }
-}
+// impl Default for Pipeline {
+//     fn default() -> Self {
+//         Pipeline {
+//             description: //"videotestsrc pattern=ball ! video/x-raw,width=640,height=480 ! videoconvert ! x264enc bitrate=5000 ! video/x-h264, profile=baseline ! rtph264pay ! udpsink host=0.0.0.0 port=5600".into(),
+//         }
+//     }
+// }
```

Comment:
No generic default string for a pipeline could be used for both UDP and RTSP, I moved it to the UDP/RTSP specific defaults.